pmoves/services/cast-tts-gateway/device_manager.py

```
@@ -0,0 +1,211 @@
+        if not force and self._last_discovery:
+            if (current_time - self._last_discovery) < self.discovery_interval:
+                return list(self.devices.values())
+
+        try:
+            # Run catt scan
+            proc = await asyncio.create_subprocess_exec(
+                "catt", "scan",
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.PIPE,
+            )
+            stdout, stderr = await proc.communicate()
```

Ruff (0.15.5)

[warning] Unpacked variable `stderr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)